aztk/config.py

```diff
@@ -218,6 +218,9 @@ def _merge_dict(self, config):
 
         if config.get('job_history_ui_port') is not None:
             self.job_history_ui_port = config['job_history_ui_port']
+        else:
+            # fall back to default port
+            self.job_history_ui_port = '18080'
```

Comment:
I'm not sure if this should go here, or near line 184, and this option would simply override it. Thoughts?

Reply:
I think this belongs on line 184.